apps/src/templates/testPages/WebLabTest.jsx

```diff
@@ -98,80 +162,81 @@ class WebLabTest extends Component {
-      <div id="main-content" className="container">
-        <div className="row">
-          <div className="col-md-6">
-            <div className="form-group">
-              <h1>Web Lab Network Support Check</h1>
-              <p>
-                Use this page to test your school's network support for our Web
-                Lab modules. Web Lab communicates with domains which may be
-                blocked by some school systems’ networks or device policies. If
-                the network check is unsuccessful, you will need to reach out to
-                your school's IT department to update your school's firewall
-                settings or device policies.
-              </p>
-              <p>
-                When checking network support, try to meet as many of these
-                criteria as possible:
-              </p>
-              <li>Use your school's internet connection</li>
-              <li>Use a student computer</li>
-              <li>Log into the computer with a student account</li>
-              <br />
-              <p>
-                Note: You do not need to be logged in to Code.org for this test
-                to be successful.
-              </p>
-              <label htmlFor="connect">
-                Click "Connect" to check whether your school's network supports
-                the Web Lab tool
-              </label>
-              <div id="status-table">
-                <table>
-                  <tbody>
-                    <tr>
-                      <th>Check</th>
-                      <th>Status</th>
-                    </tr>
-                    <tr>
-                      <td>
-                        Connected to
-                        <code>https://downloads.computinginthecore.org</code>
-                      </td>
-                      <td className="computinginthecore">
-                        {this.renderTestStatus(STATUS_COMPUTING_IN_THE_CORE)}
-                      </td>
-                    </tr>
-                    <tr>
-                      <td>
-                        Connected to <code>https://codeprojects.org</code>
-                      </td>
-                      <td className="codeprojects">
-                        {this.renderTestStatus(STATUS_CODE_PROJECTS)}
-                      </td>
-                    </tr>
-                  </tbody>
-                </table>
-              </div>
-              <div>
-                <button
-                  type="submit"
-                  id="connect"
-                  className="btn btn-primary"
-                  onClick={this.runWebLabTest}
-                >
-                  Connect
-                </button>
-              </div>
-              {this.state.renderCallToAction && this.renderCallToAction()}
-              <br />
-            </div>
+      <div className={'contentContainer'}>
+        <div className={'content'} style={styles.content}>
+          <h1>Web Lab Network Support Check</h1>
```

Comment:
do we want this page to be translatable?

Reply:
We decided not to prioritize that for now, but I filed a ticket. Will link it here!

Reply:
Linked in "Follow Up Work"!